models/particlenet/config.pbtxt

```diff
@@ -1,36 +1,39 @@
 name: "particlenet"
 platform: "onnxruntime_onnx"
-max_batch_size : 16
+max_batch_size : 160
+dynamic_batching {
+   preferred_batch_size: [ 80 ]
```

Comment:
has this choice been optimized?

Reply:
I based this on benchmarking results here: https://docs.google.com/presentation/d/1XOhrUFJN6dU_he8doJQz8cmrTgOd8NeQQ6t1PgVJE7s/edit?usp=sharing.  Throughput seems to plateau for batch sizes larger than ~80